config.json

```diff
@@ -259,7 +259,11 @@
     {
       "difficulty": 1,
       "slug": "luhn",
-      "topics": []
+      "topics": [
+	"Algorithms",
+	"Integers",
+	"Strings"
```

Comment:
I thought it might be a case of pressing a button to see what happened. We've all been there! :smile:  As far as I know it flags up someone who wants help, so if you're browsing other people's solutions you can see if anyone needs help, not sure if that will change with Nextercism.

The following exercises are also 1:
 - Leap
 - Gigasecond
 - Hamming
 - Raindrops
 - Accumulate

I intentionally left them as such when I added topics, as I thought they were appropriate ratings, to be visited later once the `config.json` was completed. I think right now, it's perhaps more important to just get difficulty ratings in, and we can adjust them once we have that baseline in place, and a better idea of what constitutes a 1. We could then perhaps articulate in words what each rating reflects, and adjust accordingly?